pytype/tools/analyze_project/config.py

```diff
@@ -84,6 +86,10 @@ def string_to_bool(s):
   return s == 'True' if s in ('True', 'False') else s
 
 
+def concate_disabled_rules(s):
+  return ','.join(t for t in s.split('\n') if t)
```

Comment:
If you do `s.split()` without any arguments, it will split on both `\s` and `\n`, which is slightly nicer.

Reply:
Done.